kart/docs.py

```diff
@@ -0,0 +1,289 @@
+"""
+This module provides functionality to
+write a man page from some given information
+about a CLI application and is inspired from
+click-contrib/click-man.
+
+https://github.com/click-contrib/click-man/blob/3a0684c9d70f94a2abef08fb3f88449dab6aea98/LICENSE
```

Comment:
```suggestion
This module provides functionality to write a man page from some given
information about a CLI application and is modified from click-man.

https://github.com/click-contrib/click-man/tree/3a0684c9d70f94a2abef08fb3f88449dab6aea98

The original click-man source is licensed under the MIT License. Changes are
licensed under the Kart license (GPL).
```

(repeated for any other files that use non-trivial click-man code)